randovania/games/am2r/json_data/The Nest.txt

```diff
@@ -504,7 +504,7 @@ Extra - map_name: rm_a6b02
   > Dock to Depths Shinespark Cave East (Bottom)
       Trivial
   > Dock to Depths Hub
-      Speed Booster and Combat (Beginner) and Shinesparking Tricks (Beginner) and Disabled Entrance Rando and Power Grip Wall
+      Speed Booster and Shinesparking Tricks (Beginner) and Disabled Entrance Rando and Power Grip Wall
```

Comment:
I think the combat was supposed to be morph glide? i dont think at least from memory that you can reach the top dock with just a shinespark, moving to the right, and then power gripping.

Reply:
The `power grip wall` covers if you land on the left platform which is trivial after the shinespark. You can then just jump to the ledge on the right and powergrip.
If you shinespark as close to the right as possible you can make the power grip with just holding right. If you use morph glide it is pretty free. Suggest I add a `OR morph glide (beginner)`

Reply:
fine for now i suppose, but in the future the room should be redone. because you can technically reach any possible door there with the shinespark and the proper knowledge.

Reply:
my thoughts too as it could have implications for entrance rando.